versions-maven-plugin/src/it/it-display-extension-updates-004/child/.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```

Comment:
Core extensions can be defined only in project root - in child module it will be skipped as I know.

Reply:
I don't see a check for `dummy-impl` so why this file exists?